docker/spark/Dockerfile

```diff
@@ -23,15 +23,20 @@ ARG PROTOBUF_VERSION=3.21.9
 COPY extensions/spark/setup/dist/ dist/
 RUN pip3 install --no-cache-dir dist/djl_spark-${DJL_VERSION}-py3-none-any.whl && \
     rm -rf dist
+RUN pip3 install --no-cache-dir pillow
```

Comment:
You can combine all RUN command together. Each `RUN` statement written in Dockerfile means each layer addtion. We would like to have as less layer as possible

Reply:
Changed